dm/dm/worker/source_worker.go

```diff
@@ -334,6 +342,13 @@ func (w *SourceWorker) EnableRelay(startBySourceCfg bool) (err error) {
 	}
 
 	// 2. initial relay holder, the cfg's password need decrypt
+	// worker's relay-dir has higher priority than source's relay-dir
+	if w.relayDir != "" {
+		workerRelayDir := filepath.Join(w.relayDir, w.name)
```

Comment:
Why do we join a worker name here 🤔

Reply:
to avoid user configurated multiple worker to use same path, in some scenario there will be multiple worker writing relay log